classes/controller/menu/class-blocks-usage-controller.php

```diff
@@ -73,19 +125,24 @@ public function plugin_blocks_report() {
 					$block_type = 'Take Action Covers - Old block';
 				}
 
-				echo '<hr>';
-				echo '<h2>' . ucfirst( str_replace( '_', ' ', $block_type ) ) . '</h2>';
-				echo '<table>';
-				echo '<tr style="text-align: left"><th>ID</th><th>Title</th></tr>';
-				foreach ( $results as $result ) {
-					echo '<tr><td>'.$result->ID .'</td>';
-					echo  '<td><a href="post.php?post=' . $result->ID . '&action=edit" >' . $result->post_title . '</a></td></tr>';
+				if ( 'text' === $type ) {
+					echo '<hr>';
+					echo '<h2>' . ucfirst( str_replace( '_', ' ', $block_type ) ) . '</h2>';
+					echo '<table>';
+					echo '<tr style="text-align: left">
+							<th>' . __( 'ID', 'planet4-blocks-backend' ) . '</th>
+							<th>' . __( 'Title', 'planet4-blocks-backend' ) . '</th>
+					</tr>';
+					foreach ( $results as $result ) {
+						echo '<tr><td>'.$result->ID .'</td>';
+						echo  '<td><a href="post.php?post=' . $result->ID . '&action=edit" >' . $result->post_title . '</a></td></tr>';
+					}
+					echo '</table>';
+				} else {
+					$report[ ucfirst( str_replace( '_', ' ', $block_type ) ) ] = count($results);
```

Comment:
What is the reason for using ucfirst on the block type here?

Reply:
The API fields(block type name) display as a header title in Spreadsheet report.